presto-raptor/src/test/java/com/facebook/presto/raptor/storage/TestShardWriter.java

```diff
@@ -250,30 +250,6 @@ public void testWriter(boolean useOptimizedOrcWriter)
         }
     }
 
-    @SuppressWarnings("EmptyTryBlock")
-    @Test
-    public void testWriterZeroRows()
```

Comment:
why was this test removed?

Reply:
Short answer, because that is what Trino did https://github.com/trinodb/trino/commit/238f4b6bb1ac98130cff7bee88d5e622e5a2808f#diff-82a51d3cf239ee341a4eab63c8c88c82367b389a9652cbaa067631d131a60930

Long answer... I don't know. I'll bring it back and see what happens when I run it.

Reply:
The test still passes. So I guess we should keep it!